wp-content/themes/Belladonna/scss/main.scss

```diff
@@ -362,47 +363,47 @@ h1.page-title {
 	}
 }
 
-h1.page-title.Green {
+h1.page-title.Green, .block_image_left h2.Green {
 	color: $green;
 }
-h1.page-title.Blue {
+h1.page-title.Blue, .block_image_left h2.Blue {

```

Comment:
You can change all these to use hook classes. So for example `.u-color--blue` and `.u-background-color--blue` and apply the classes to each of the php elements. If the colors are pulled in conditionally, use php to transform it into the proper class format.